src/raw/request.rs

```diff
@@ -91,23 +168,3 @@ impl Borrow<str> for Empty {
         unreachable!("Should never be used as a value")
     }
 }
-
```

Comment:
Those lines are just moved above due to rebase

Reply:
Did you want me to add the line back?

Reply:
I think it's OK - it's probably for the best.